www/source/partials/docs/_glossary-launcher.html.md.erb

```diff
@@ -0,0 +1,23 @@
+# <a name="glossary-launcher" id="glossary-launcher" data-magellan-target="glossary-launcher">Launcher</a>
+
+Habitat's Launcher is a sidecar process for the Supervisor which provides a mechanism for launching processes on behalf of the Supervisor. It is the entry point for running the Supervisor and is the Supervisor for the Supervisor. Whereas the Supervisor is able to automatically update itself, the Launcher is currently released a bit differently, by design; it should be rare that the Launcher ever needs to change.
+
+To update your Launchers, you will need to run
+
+```
+hab pkg install core/hab-launcher
+```
+
+on each machine that has a Supervisor, and then restart your Supervisor. This will, by necessity, require a restart of supervised services, so factor that into your planning.
+
+The Launcher is designed to run as process 1; it's extremely minimal by design. It's responsibilities are to:
+
+* Launch the Habitat Supervisor
+* Restart the Habitat Supervisor on shutdown (Supervisor for the Supervisor)
+* Start, Stop, and Restart processes on behalf of the Supervisor
+
+Launcher is specifically designed to enable the Supervisor to self update without shutting down or re-parenting the services that the Supervisor is supervising. It is versioned separately from the Supervisor and should be updated very infrequently since an update of the Launcher could require a system restart if it is running as process 1.
+
+The Supervisor is *always* started by the Launcher. The Launcher will always attempt to start the latest version of the Supervisor package installed on the host. When the Launcher starts the Supervisor a handshake is made over a platform specific [IPC](https://en.wikipedia.org/wiki/Inter-process_communication) channel. The Supervisor communicates with the Launcher with a binary protocol through the Launcher client.
```

Comment:
```suggestion
The Launcher *always* starts the Supervisor. The Launcher always attempts to start the most recent Supervisor package version installed on the host. When the Launcher starts the Supervisor, it makes a handshake over a platform specific [IPC](https://en.wikipedia.org/wiki/Inter-process_communication) channel. The Supervisor communicates with the Launcher with a binary protocol through the Launcher client.
```

Reply:
Latest is stronger wording for "most recent version" here and should be kept.

Sidenote: The "latest" language comes from git itself, which refers to a particular commit hash.